addons/plotting/functions/fnc_drawLine.sqf

```diff
@@ -0,0 +1,63 @@
+#include "script_component.hpp"
+/*
+ * Authors: Timi007
+ * Draws a line plot in 3D or on the map. Must be called every frame.
+ *
+ * Arguments:
+ * 0: Start position ASL <ARRAY>
+ * 1: End position ASL <ARRAY>
+ * 2: Visual properties <ARRAY>
+ *      0: Icon <STRING>
+ *      1: Color RGBA <ARRAY>
+ *      2: Scale <NUMBER>
+ *      3: Angle <NUMBER>
+ *      4: Line width <NUMBER>
+ * 3: Formatters <ARRAY>
+ *      0: Distance formatter <CODE>
+ *      1: Azimuth formatter <CODE>
+ * 4: Camera position ASL <ARRAY> (default: Don't check render distance)
+ * 5: Map control <CONTROL> (default: Draw in 3D)
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [[0, 0, 0], [100, 100, 0], ["", [1, 0, 0, 1], 1, 0, 5], [{_this toFixed 0}, {_this toFixed 1}]] call zen_plotting_fnc_drawLine
+ *
+ * Public: No
+ */
+
+params ["_startPos", "_endPos", "_visualProperties", "_formatters", ["_ctrlMap", controlNull, [controlNull]]];
+_visualProperties params ["_icon", "_color", "_scale", "_angle", "_lineWidth"];
+
+private _distance = _startPos vectorDistance _endPos;
+
+private _azimuthToStart = _endPos getDir _startPos;
+private _azimuthToEnd = _startPos getDir _endPos;
```

Comment:
Should this be the relative direction?

Reply:
This is the direction from start to end pos and end to start. Both `_startPos` and `_endPos` are PosASL. I don't understand the relative part in your question.

Reply:
IIRC, when attached to objects the direction value being displayed did not seem intuitive. If that makes sense.

Reply:
Need to check what you mean. It should show the same value as the unit just opening their compass.

Reply:
I guess it just comes to whether we want to know that the position is 90 degrees relative to the vehicle or just the difference in their bearings. The former what what I assumed the value was.

Reply:
What purpose does relative bearing w.r.t. the attached unit have to the Zeus or any unit on the ground? I understand if you want the direction as a relative clock, but I have never given or received relative bearing. For artillery corrections, you also don't use relative directions.